src/components/user-message/user-message.component.jsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { List, Image } from "semantic-ui-react";
+
+import "./user-message.styles.scss";
+
+const UserMessage = (props) => {
+  const { message } = props;
```

Comment:
Maybe we could destruct props on the function parameters?
```js
const UserMessage = ({ message }) => (
  ...
)
```

What do you think?